pom.xml

```diff
@@ -33,6 +33,7 @@
     <module>features</module>
     <module>plugins</module>
     <module>third_party</module>
+    <module>eclipse</module>
```

Comment:
So the difference will be that now we build `eclipse/neon`, `eclipse/oxygen`, and `eclipse/photon` by default at the top level (of course except `eclipse/ide-target-platform`)?

Reply:
We were already pulling in these three projects when importing with our previous instructions.  They will still be found and selected by default under the new instructions, though we will now say to unselect them.

Reply:
I should note that pulling in `eclipse/{neon,oxygen,photon}` by default adds a negligible overhead.  It's `eclipse/ide-target-platform` that takes a while:
```
[INFO] gcp-eclipse-neon 4.6.0-SNAPSHOT .................... SUCCESS [  0.003 s]
[INFO] gcp-eclipse-oxygen 4.7.0-SNAPSHOT .................. SUCCESS [  0.002 s]
[INFO] gcp-eclipse-photon 4.8.0-SNAPSHOT .................. SUCCESS [  0.002 s]
[INFO] ide-target-platform.repo ........................... SUCCESS [01:02 min]
[INFO] eclipse-setup 0.1.0-SNAPSHOT ....................... SUCCESS [  0.002 s]
```